include/HUDL/dev/LED.hpp

```diff
@@ -7,41 +7,38 @@ namespace EVT::core::DEV {
 
 class LED {
```

Comment:
I'm pretty sure we have an LED class in EVT-core. Do you know why this is here?

Reply:
Unknown why it was there in the first place. Looks like it was not reference in CMake, so it is now removed.